geoportailv3/static/js/maincontroller.js

```diff
@@ -86,6 +90,16 @@ app.MainController.prototype.setMap_ = function() {
 
 
 /**
+ * @private
+ */
+app.MainController.prototype.updateSelectedLayers_ = function() {
+  var items_ = this['map'].getLayers().getArray().slice().reverse();
+  items_.pop();
+  this['selectedLayers'] = items_;
```

Comment:
I also note that we will need a more complex filter when we'll use vector layers for drawing etc.